### PR TITLE
Updated definitions

### PR DESCRIPTION
**Pull request by ektravel**
Updated definitions in PUT /wishlists/{wishlist_id}

### CODE REVIEW

File: reference/wishlists.v3.yml [hunk 1]
```diff
@@ -814,47 +791,29 @@ securityDefinitions:
...
       X-Auth-Token: {access_token}
-
       ```
 
-
-      * For more information on Authenticating BigCommerce APIs, see:
-      [Authentication](https://developer.bigcommerce.com/api-docs/getting-started/authentication). 
+      * For more information on Authenticating BigCommerce APIs, see: [Authentication](https://developer.bigcommerce.com/api-docs/getting-started/authentication). 
 security:
   - X-Auth-Client: []
```
Comment (traci707) [inline reference/wishlists.v3.yml:818]: Is X-Auth-Client needed here?

**Review by traci707 — COMMENTED** [review]
Everything looks fine. I just wondered if you needed X-Auth-Client.

File: reference/wishlists.v3.yml (hunk 1 above)
Comment (ektravel) [reply reference/wishlists.v3.yml:818]: I believe this field is generated by Stoplight. X-Auth-Client is not listed in securityDefinitions, so I think we should be good.

**Comment by ektravel** [comment]
Changed capitalization, updated overview, updated customer_id.

File: reference/wishlists.v3.yml (hunk 1 above)
Comment (traci707) [reply reference/wishlists.v3.yml:843]: thanks for checking

File: reference/wishlists.v3.yml [hunk 2]
```diff
@@ -814,47 +816,29 @@ securityDefinitions:
...
       Content-Type: application/json
-
       X-Auth-Token: {access_token}
-
       ```
 
-
-      * For more information on Authenticating BigCommerce APIs, see:
-      [Authentication](https://developer.bigcommerce.com/api-docs/getting-started/authentication). 
+      * For more information on Authenticating BigCommerce APIs, see: [Authentication](https://developer.bigcommerce.com/api-docs/getting-started/authentication).
```
Comment (traci707) [inline reference/wishlists.v3.yml:841]:
```suggestion
      * For more information on Authenticating BigCommerce APIs, see [Authentication](https://developer.bigcommerce.com/api-docs/getting-started/authentication). 
```

File: reference/wishlists.v3.yml [hunk 3]
```diff
@@ -484,7 +493,7 @@ definitions:
         type: string
         example: School Shopping
       items:
-        description: Array of Wishlist items.
+        description: Array of wishlist items.
         type: array
         items:
           title: Add Item to Wishlist
```
Comment (traci707) [inline reference/wishlists.v3.yml:499]:
```suggestion
          title: Add item to wishlist
```

File: reference/wishlists.v3.yml [hunk 4]
```diff
@@ -17,15 +40,11 @@ produces:
...
-
+      description: |-
+        Creates a wishlist and wishlist item. More than one item can be added in the POST.
 
         **Required Fields**
-
         * name
-
         * customer_id
       summary: Create a Wishlist
```
Comment (traci707) [inline reference/wishlists.v3.yml:49]:
```suggestion
      summary: Create a wishlist
```

File: reference/wishlists.v3.yml [hunk 5]
```diff
@@ -92,9 +111,7 @@ paths:
               type:
                 type: string
     get:
-      description: >-
-        Returns a list of Wishlists. Optional filter parameters can be passed
-        in.
+      description: Returns a list of wishlists. Optional filter parameters can be passed in.
       summary: Get All Wishlists
```
Comment (traci707) [inline reference/wishlists.v3.yml:115]:
```suggestion
      summary: Get all wishlists
```

File: reference/wishlists.v3.yml [hunk 6]
```diff
@@ -158,7 +171,7 @@ paths:
                 type: string
   '/wishlists/{wishlist_id}/items/{item_id}':
     delete:
-      description: Deletes a Wishlist Item.
+      description: Deletes a wishlist item.
       summary: Delete Wishlist Item
```
Comment (traci707) [inline reference/wishlists.v3.yml:175]:
```suggestion
      summary: Delete wishlist item
```

File: reference/wishlists.v3.yml [hunk 7]
```diff
@@ -226,7 +239,7 @@ paths:
         required: true
   '/wishlists/{wishlist_id}':
     get:
-      description: Returns a single Wishlist.
+      description: Returns a single wishlist.
       summary: Get a Wishlist
```
Comment (traci707) [inline reference/wishlists.v3.yml:243]:
```suggestion
      summary: Get a wishlist
```

File: reference/wishlists.v3.yml [hunk 8]
```diff
@@ -279,16 +292,12 @@ paths:
...
-        wishlist name, customer ID or if the wishlist is available publicly.
+      description: |-
+        Updates a wishlist.
 
+        Use this endpoint to update existing wishlist items, change the wishlist's name and whether the wishlist is available publicly. To add or delete a wishlist item, see [Wishlist Items](https://developer.bigcommerce.com/api-reference/store-management/wishlists/wishlists-items).
 
         **Required Fields**
-
         * customer_id
       summary: Update a Wishlist
```
Comment (traci707) [inline reference/wishlists.v3.yml:302]:
```suggestion
      summary: Update a wishlist
```

File: reference/wishlists.v3.yml [hunk 9]
```diff
@@ -354,7 +363,7 @@ paths:
               type:
                 type: string
     delete:
-      description: Deletes a Wishlist.
+      description: Deletes a wishlist.
       summary: Delete a Wishlist
```
Comment (traci707) [inline reference/wishlists.v3.yml:367]:
```suggestion
      summary: Delete a wishlist
```

File: reference/wishlists.v3.yml [hunk 10]
```diff
@@ -437,7 +446,7 @@ paths:
         '500':
           description: Internal server error.
       summary: Add Wishlist Item
```
Comment (traci707) [inline reference/wishlists.v3.yml:448]:
```suggestion
      summary: Add wishlist item
```

File: reference/wishlists.v3.yml [hunk 11]
```diff
@@ -517,24 +526,24 @@ definitions:
         description: The title of the wishlist.
         type: string
       items:
-        description: Array of Wishlist items.
+        description: Array of wishlist items.
         type: array
         items:
           title: Wishlist Item
```
Comment (traci707) [inline reference/wishlists.v3.yml:532]:
```suggestion
          title: Wishlist item
```

**Review by traci707 — COMMENTED** [review]
For consistency, I tried to ensure "wishlist" was lowercase throughout.

File: reference/wishlists.v3.yml (hunk 4 above)
Comment (ektravel) [reply reference/wishlists.v3.yml:49]: Changing the summary field will change the way the name of the endpoint appears on the left pane of API Reference.

File: reference/wishlists.v3.yml (hunk 5 above)
Comment (ektravel) [reply reference/wishlists.v3.yml:115]: Changing the summary field will change the way the name of the endpoint appears in the left pane of API Reference.

File: reference/wishlists.v3.yml (hunk 7 above)
Comment (ektravel) [reply reference/wishlists.v3.yml:243]: Changing the summary field will change the way the name of the endpoint appears in the left pane of API Reference.

File: reference/wishlists.v3.yml (hunk 8 above)
Comment (ektravel) [reply reference/wishlists.v3.yml:302]: Changing the summary field will change the way the name of the endpoint appears in the left pane of API Reference.

File: reference/wishlists.v3.yml (hunk 9 above)
Comment (ektravel) [reply reference/wishlists.v3.yml:367]: Changing the summary field will change the way the name of the endpoint appears in the left pane of API Reference.

File: reference/wishlists.v3.yml (hunk 10 above)
Comment (ektravel) [reply reference/wishlists.v3.yml:448]: Changing the summary field will change the way the name of the endpoint appears in the left pane of API Reference.

**Pull request merged**